test/scss/_errors.scss

```diff
@@ -0,0 +1,34 @@
+// Errors Test
+// ===============
+
+$not-a-color: #ggg;
+
+@mixin color-mixin($color) {
+  @if (type-of($color) != color) {
+    @error 'Must pass a valid color';
+  }
+
+  .foo {
+    color: $color;
+  }
+}
+
+@include test-module('Assert Error') {
+  @include test('Detects when an error is thrown') {
+    @include assert-error() {
+      @include color-mixin($not-a-color);
+    }
+  }
+
+  @include test('Compares a partial message to the emitted message (case-insensitive)') {
+    @include assert-error('vAliD coLOr') {
+      @include color-mixin($not-a-color);
+    }
+  }
+
+  @include test('Compares the full message to the emitted message (case-insensitive)') {
+    @include assert-error('mUst PasS a vAliD cOloR') {
```

Comment:
I can't get these tests to fail.

- I commented out the error above, and it still passed
- I changed the message completely, and it still passed

I'd also want to discuss more if case-insensitivity and partial-matches are the right way to go. We don't support the former anywhere else in True, and we only support the latter when explicitly requested using the `contains()` mixin.

Reply:
Thanks. I figured this would be a point of discussion. I think I would prefer then to see something for different scenarios:
- `assert-throws()`: passes if any error is thrown
- `assert-error-matches(msg)`: passes if the msg equals the thrown error message
- `assert-error-contains(msg)`: passes if the msg appears within the thrown error message